packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/repositories/auth_repository.dart

```diff
@@ -44,8 +46,9 @@ class AuthRepository {
     try {
       // TODO: Try to fetch new access token using refreshToken and save it
       // https://flutteragency.com/refresh-token-using-interceptor-in-dio/
+      return '';
     } catch (e) {
-      print(e.toString());
+      debugPrint(e.toString());
```

Comment:
Why we use debugPrint instead of `log` ?

Reply:
By default the `debugPrint` does the same thing as `print` or `log`. But the advantage of using `debugPrint` over the other two is that it can be overriden. I've included the following code that is called when the app is run, so we don't accidentally leak any data (in staging, prod or any non-dev environment) when we log it.

```dart
void _disableLogs(EnvironmentConfig environment) {
  // Disable logs for non-dev environments
  if (environment == EnvironmentConfig.dev) return;

  debugPrint = (String? message, {int? wrapWidth}) {};
}
```

This will allow the developers to easily log any message while developing the app, while not having to worry about app logs being collected by some malicious sources.